app/com/mohiva/play/htmlcompressor/HTMLCompressorFilter.scala

```diff
@@ -45,7 +47,7 @@ abstract class HTMLCompressorFilter extends CompressorFilter[HtmlCompressor] {
  * @param configuration The Play configuration.
  * @param environment The Play environment.
  */
-class DefaultHTMLCompressorFilter @Inject() (val configuration: Configuration, environment: Environment)
+class DefaultHTMLCompressorFilter @Inject() (val configuration: Configuration, environment: Environment)(implicit val mat: Materializer)
```

Comment:
I think Guice cannot handle the implicit `Materializer` here. For what is the `Materializer` needed here?